### ISSUE TITLE
Closing Visual Scenario Editor results in Segmentation Fault

### ISSUE THREAD

**Issue by krtt** [lgsvl/simulator#1750]
Sometimes when running the visual scenario editor, it seems to need too much resources and either hangs or results in segmentation fault after closing VSE from the "exit" menu button. What can be wrong or what can I improve?

Sim v2021.2
Ubuntu 18.04
Dell XPS 15 9570, 32GB RAM
NVIDIA GeForce GTX 1050 Ti, 8GB memory, Driver Version: 470.63.01

[Player.log](https://github.com/lgsvl/simulator/files/7150047/Player.log)

**Comment by EricBoiseLGSVL** [comment]
Yep, log states out of memory. This really depends on the map you are using and resources available.  Is it a large map?

I do see that your GPU is pretty low.  I recommend getting a better one if you are doing any HDRP development.  Note, using Unity Editor and simulation uses significantly more resources than running a binary which we list min spec as a 1080 GTX.
